internal/services/appservice/linux_function_app_on_container_resource_test.go

```diff
@@ -0,0 +1,216 @@
+package appservice_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonids"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type LinuxFunctionAppOnContainerResource struct{}
+
+func TestAccLinuxFunctionAppOnContainer_basicDocker(t *testing.T) {
```

Comment:
seems `storage_key_vault_secret_id, site_config.application_insights_key, site_config.elastic_instance_minimum ,site_config.linux_fx_version, identity` are not test. Could you add these testcases?